test/Sentry.Tests/SerializationTests.Serialization_TransactionAndSpanData.verified.txt

```diff
@@ -0,0 +1,44 @@
+﻿{
+  EventId: {},
+  SpanId: {},
+  ParentSpanId: {},
+  Name: name,
+  Platform: csharp,
+  StartTimestamp: DateTimeOffset_1,
+  Operation: operation,
+  Description: ,
+  IsSampled: false,
+  SampleRate: 0.0,
+  Request: {},
+  Contexts: {
+    trace: {
+      SpanId: {},
+      ParentSpanId: {},
+      Operation: operation,
+      Description: ,
+      IsSampled: false,
+      Extra: {
+        transaction1: transaction_value
+      }
+    }
+  },
+  User: {},
+  Sdk: {},
+  Extra: {
+    transaction1: transaction_value
+  },
+  Spans: [
+    {
+      SpanId: {},
+      ParentSpanId: {},
+      StartTimestamp: DateTimeOffset_2,
+      IsFinished: false,
+      Operation: childop,
+      IsSampled: false,
+      Extra: {
```

Comment:
Same again - I think this is suppose to be serialised as a data property on spans now right?

Reply:
I've moved the tests to just json path walk to ensure the tree is serialized properly